docs/remote/containers-advanced.md

```diff
@@ -537,25 +537,30 @@ Once the needed CLIs are in place, you can also work with the appropriate contai
 
 See the following example dev containers definitions for additional information on a specific scenario:
 
-* [Docker-from-Docker](https://aka.ms/vscode-remote/samples/docker-from-docker) - Includes the Docker CLI and illustrates how you can use it to access your local Docker install from inside a dev container by volume mounting the Docker Unix socket.
+* [Docker-in-Docker](https://aka.ms/vscode-remote/samples/docker-in-docker) - Illustrates how to run Docker (or Moby) entirely inside a container. Provides support for bind mounting all folders inside the development container, but cannot reuse your local machine's cache.
```

Comment:
https://aka.ms/vscode-remote/samples/docker-in-docker isn't loading for me:

![image](https://user-images.githubusercontent.com/25310137/110992820-4e92ad80-832b-11eb-9687-4f18f2926a93.png)